### ISSUE TITLE
ability to set a callback for drop event

### ISSUE THREAD

**Issue by leedo** [mephraim/raphael.draggable#3]
Some mechanism to add a callback for the drop event would be helpful.

**Comment by mephraim** [comment]
I agree. I'd like to start working on adding some events that match up with the HTML5 drag and drop events (http://www.whatwg.org/specs/web-apps/current-work/multipage/dnd.html#dnd). Thoughts?
